src/main/java/oshi/software/os/windows/WindowsFileSystem.java

```diff
@@ -11,20 +11,17 @@
  * Maintainers:
  * dblock[at]dblock[dot]org
  * widdis[at]gmail[dot]com
+ * enrico[dot]bianchi[at]gmail[dot]com

```

Comment:
This header is auto-overwritten by maven commit/release code.  Replace my name with yours in the @author field below instead.